unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TeleportPromptHUD/TeleportPromptHUDController.cs

```diff
@@ -211,7 +228,7 @@ internal class TeleportData
         public Vector2Int? coordinates;
 
         public Vector2Int GetCoordinates() =>
-            (coordinates ??= new Vector2Int(int.TryParse(destination.Split(',')[0], out int x) ? x : 0,
+            (coordinates ?? new Vector2Int(int.TryParse(destination.Split(',')[0], out int x) ? x : 0,
```

Comment:
why was this change introduced?
seems `??=` was to avoid parsing `destination` string every time `GetCoordinates` is called

Reply:
uhmm true,  must've slipped while testing stuff, I'll revert that change just in case.